packet_analysis/sql/sql_insert.py

```diff
@@ -0,0 +1,60 @@
+import sqlite3
+from sqlite3 import Error
+
+# All data access objects from __inti__.py -> __all__
+from .dao import *
+
+# Bindings for table creation SQL
+table_bindings = {
+    "ARP": "CREATE TABLE IF NOT EXISTS ARP (\n              {0});".format(arp.table_sql()),
+    "DHCP": "CREATE TABLE IF NOT EXISTS DHCP (\n              {0});".format(dhcp.table_sql()),
+    "DNS": "CREATE TABLE IF NOT EXISTS DNS (\n              {0});".format(dns.table_sql()),
+    "EAPOL": "CREATE TABLE IF NOT EXISTS EAPOL (\n              {0});".format(eapol.table_sql()),
+    "Ethernet": "CREATE TABLE IF NOT EXISTS Ethernet (\n              {0});".format(ethernet.table_sql()),
+    "IP":  "CREATE TABLE IF NOT EXISTS IP (\n              {0});".format(ip.table_sql()),
+    "NTP": "CREATE TABLE IF NOT EXISTS NTP (\n              {0});".format(ntp.table_sql()),
+    "TCP": "CREATE TABLE IF NOT EXISTS TCP (\n              {0});".format(tcp.table_sql()),
+    "UDP": "CREATE TABLE IF NOT EXISTS UDP (\n              {0});".format(udp.table_sql()),
+}
+
+def create_connection(path):
```

Comment:
Isn't this the same function as in `sql_helper`?

Reply:
That it is, it's not even used anywhere. Must have been copy-pasta, removed.